llms/mlx_lm/tuner/utils.py

```diff
@@ -31,6 +31,117 @@ def build_schedule(schedule_config: Dict):
     else:
         return bound_schedule_fn
 
+def linear_to_dora_layers(
+    model: nn.Module,
+    num_dora_layers: int,
+    config: Dict,
+):
```

Comment:
Instead of copying the function `linear_to_lora_layers` I would just pass the bool `use_dora` to `linear_to_lora_layers`.

Reply:
remove  `linear_to_dora_layers`  and  pass `use_dora` to `linear_to_lora_layers`